handbook/FileService-MinIO/FileService-MinIO.md

```diff
@@ -0,0 +1,59 @@
+# Leverage Cloud File Storage with MinIO
+
+SystemLink File Service allows you to configure and use the Amazon S3 cloud storage instead of a file share. While NI officially addresses the Amazon S3 cloud storage, the File Service also works with MinIO, a server-side software storage stack that is compatible with Amazon S3.
+
+To use MinIO with SystemLink, you need SystemLink version 2020 R3 or later.
+
+To use MinIO as a storage provider, set up the MinIO server on a system that you would like to upload files to. This can be the same machine as the SystemLink server or an own dedicated server. Then, configure the File Service to use that server.
+
+## Running MinIO server
+
+1. Download the server application from the [MinIO website](https://min.io/download).
+
+2. From the command line, run the following command to run the server:
+
+    ```bash
+    minio server <PathToRootFolder>
+    ```
+
+    <figure>
+    <img src="../../img/minio-server.png" width="500" />
+    <figcaption>Running a minio server.</figcaption>
+    </figure>
+
+3. When starting the MinIO server for the first time, use a web browser to connect to the endpoint that the command line prints out. Create a bucket by using the `+` button in the right bottom corner.
```

Comment:
```suggestion
3. When starting the MinIO server for the first time, use a web browser and navigate to the URL that the command line prints out. Create a bucket by clicking the `+` button in the right bottom corner.
```

The terminology used in this context might be a bit too in the weeds for a IT admin wanting to set this up. I suggest we put things more in layman's terms.

After the point about navigating in a browser, maybe we should add a note about what the user should expect to see at this point?

Reply:
>After the point about navigating in a browser, maybe we should add a note about what the user should expect to see at this point?
The sentence would be along the line:
>Now you'll see the bucket in the web UI.

There isn't anything spectacular happening, is this of any help to the user?

Reply:
Thanks for suggesting better phrasing, btw. :)